plio/cache.py

```diff
@@ -0,0 +1,34 @@
+from django.core.cache import cache
+
+
+def get_cache_key(instance):
+    """Calculates cache key for an instance"""
+    instance_class = instance.__class__.__name__
+    return {
+        "Video": f"video_{instance.pk}",
```

Comment:
Upper case is the format you guys have decided?

Reply:
this is like a switch-case. I'm matching the class name for the instance and using it to determine the key. For a video instance, the class name is `Video` defined inside `plios/models.py`, hence it needs to match.

Do you think it'll be better to transform instance_class to lowercase and then match?